portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,33 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private ArrayList<String> messages;
+
+  @Override
+  public void init() {
+    this.messages = new ArrayList<>();
+
+    messages.add("Hello. I am a string.");
+    messages.add("This is another string.");
+    messages.add("I too am a string.");
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Amy!</h1>");
+    String json = convertToJsonUsingGson(messages);
+
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  /**
```

Comment:
This entire comment can be made more simple like this:

Converts {@code toConvert} into a JSON string using GSON.

@param is only necessary if the summary doesn't make it clear what the param is for.